### ISSUE TITLE
How could I segment object without need to resize the whole image for it

### ISSUE THREAD

**Issue by VictorGimenez** [keras-team/keras-cv#2457]
I am following the example from the official webpage to use Segment Anything tool: `https://keras.io/guides/keras_cv/segment_anything_in_keras_cv/` but I am using a different image from the example (1200, 1800, 3) where my image has (1920,1080) and there are another ones with percentual reduction from this one of (1920, 1080) and I am trying to maintain the same width and height from the input image to have the real contours/segmentation coordinates (from the original image) and to establish the input_point and input_label on the original resolution (width, height) without the need to resize through the inference_resizing method.

I have tried to replace 1024 by another width values such as 1920, 1728, 1536... or suppose that I have a "x" value it would be this "x" value instead this 1024 but I always get error on the outputs dictionary where I set the "images", "points" and "labels" dicts.

What should I do to maintain the same width, height to segment my image to have the real x and y coordinates used to segment my object?

**Comment by sachinprasadhs** [comment]
Hi, 

Segment anything has been made available in Keras Hub and it's detailed usage can be found here https://keras.io/keras_hub/guides/segment_anything_in_keras_hub/
If you're still facing an issue, you can close this issue and open a new issue in `/keras-hub`
